policyengine_us/parameters/gov/states/ar/tax/income/credits/inflationary_relief/max_amount.yaml

```
@@ -0,0 +1,27 @@
+description: Arkansas allows for this maximum inflation relief credit amount, based on filing status.
```

```suggestion
description: Arkansas allows for this maximum inflationary relief tax credit amount, based on filing status.
```
This is the name in the worksheet so use it everywhere